mubench.reviewsite/setup/schema_util.php

```diff
@@ -0,0 +1,162 @@
+<?php
```

Comment:
I think I would prefer a filename like `database_setup_utils`... not perfect, but it's not a schema utils anymore.